ansible/roles/packages/tasks/repo-redhat.yaml

```diff
@@ -1,4 +1,49 @@
 ---
+
+- name: install epel-repository
+  when:
+    - not offline_mode_enabled
+  block:
+    - name: Remove appstream repository
+      yum_repository:
+        name: appstream
+        state: absent
```

Comment:
Maybe we need to update the cache?
```
fatal: [default]: FAILED! => {"changed": false, "msg": "Failed to download metadata for repo 'appstream': Cannot prepare internal mirrorlist: No URLs in mirrorlist", "rc": 1, "results": []}
```

https://teamcity.mesosphere.io/buildConfiguration/ClosedSource_KonvoyImageBuilder_E2eCentOS8/3341768?buildTab=log&focusLine=1336&linesState=469

Reply:
removed this bit. instead of adding it as "disabled" repo.